packages/langium/test/grammar/type-system/type-validator.test.ts

```diff
@@ -78,4 +78,20 @@ describe('validate params in types', () => {
         expect(diagnostics.filter(d => d.severity === DiagnosticSeverity.Error)).toHaveLength(0);
     });
 
+});
+
+describe('Inferred Types Validation', () => {
```

Comment:
To keep consistency with the previous test group, I suggest to call it "validate inferred types"

Reply:
Yep, can do